lib/datasource/index.ts

```diff
@@ -37,7 +50,14 @@ async function fetchReleases(
     logger.warn('Unknown datasource: ' + datasource);
     return null;
   }
-  const dep = await (await load(datasource)).getPkgReleases(config);
+  const registryUrls = await resolveRegistryUrls(
+    datasource,
+    config.registryUrls
+  );
+  const dep = await (await load(datasource)).getPkgReleases({
+    ...config,
+    registryUrls,
+  });
```

Comment:
Why not first await datasource and then passing to `resolveRegistryUrls` ?

Reply:
Can you explain more? I don't follow

Reply:
Got it, I'll fix